src/Compilers/CSharp/Test/Symbol/Symbols/AnonymousTypesSemanticsTests.cs

```diff
@@ -564,6 +925,98 @@ public static void Test1(int x)
         }
 
         [Fact()]
+        public void AnonymousTypeSymbols_Error_Simple_OperationTree()
+        {
+            string source = @"
+class ClassA
+{
+    public static void Test1(int x)
+    /*<bind>*/{
+        object v1 = new
+        {
+            aa = xyz,
+            BB = """",
+            CCC = new SSS()
+        };
+
+        object v2 = new
+        {
+            aa = new SSS(),
+            BB = 123.456,
+            CCC = new
+            {
+                (new ClassA()).aa,
+                ClassA.BB,
+                ClassA.CCC
+            }
+        };
+    }/*</bind>*/
+}
+";
+            string expectedOperationTree = @"
+IBlockStatement (2 statements, 2 locals) (OperationKind.BlockStatement, IsInvalid) (Syntax: '{ ... }')
```

Comment:
Same issue as others - this flag is current based off BoundNode.HasErrors. @heejaechang is planning to replace this with an IsInValid extension method.

---
In reply to: [123879687](https://github.com/dotnet/roslyn/pull/20364#discussion_r123879687) [](ancestors = 123879687)